src/basic/INDEXLIST.cpp

```diff
@@ -48,7 +48,10 @@ INDEXLIST::INDEXLIST(const RunParams& params)
 
   setVariantDefined( Base_CUDA );
 
-  setVariantDefined( Base_HIP );
+  if (0) {
```

Comment:
😞

Reply:
I agree. I could add an option to the code instead of if 0. How about something like one of these:
--allow-deadlocking-implementations
--allow-incorrect-implementations
--allow-questionable-implementations

Reply:
Is this a new issue (e.g., specific to rocm versions(s)) or has it always been there? Should we report this to be fixed?

Dunno what to call it... '--allow-problematic-implementations'?

Reply:
No this has been an issue for as long as that kernel has been around with a hip version.

Reply:
Added an option to allow _problematic_ implementations to run.